rogue-thi-app/public/locales/de/common.json

```diff
@@ -17,6 +17,12 @@
                 "showReimannsStatic": "Reimanns anzeigen (feste Gerichte)",
                 "showCanisius": "Canisiuskonvikt anzeigen"
             },
+            "languageFood":{
+                "title":"Sprache für den Speiseplan",
+                "german":"Deutsch",
```

Comment:
We already habe the translated Lang names in the [`languages.json`](https://github.com/neuland-ingolstadt/neuland.app/blob/df2131b72550aa3b445262fc4449983f7920740e/rogue-thi-app/data/languages.json#L3). Therefore, we shouldn't redefine them here, especially when we add new languages.